packages/support/resources/views/components/section/index.blade.php

```diff
@@ -62,7 +63,7 @@
                 'cursor-pointer' => $collapsible,
                 match ($compact) {
                     true => 'px-4 py-2.5',
-                    false => 'px-6 py-4',
+                    false => ! $contained ? 'py-4' : 'px-6 py-4',
```

Comment:
Could you split these into separate array items too?
Also, is the vertical padding needed when uncontained?